templates/dex.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       kubeaddons.mesosphere.io/provides: ingresscontroller
   chartReference:
     chart: dex
-    repo: https://mesosphere.github.io/charts/stable
+    repo: https://timaa2k.github.io/charts
```

Comment:
let's use `https://mesosphere.github.io/charts/stable` and bump the dex chart version